tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -207,6 +207,39 @@ test.describe( 'Nested Tabs tests', () => {
 		await expect( activeTab ).toHaveCSS( 'margin-bottom', '50px' );
 		await expect( lastTab ).toHaveCSS( 'margin-top', '25px' );
 	} );
+
+	test( 'Check that active and non-active tabs color changes on hover', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		await wpAdmin.setExperiments( {
+			container: true,
+			'nested-elements': true,
+		} );
+
+		const editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		// Add widgets.
+		await editor.addWidget( 'nested-tabs', container );
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tab-title.e-normal.e-active' );
+
+		// Act.
+		await editor.activatePanelTab( 'style' );
+		await page.locator( '.elementor-control-section_title_style' ).click();
```

Comment:
@davseve There is a background color selector inside the `editor-page.js` file.

Reply:
Hi, `setBackgroundColor()` sets the background color.
Here I choose a color and not the background color.
But great function :) .
If I didn't understand you well, let's talk about that.

Reply:
@davseve My mistake. You are correct.